frontend/src/metabase-lib/util.ts

```diff
@@ -0,0 +1,10 @@
+import * as ML from "cljs/metabase.lib.js";
+import type { DatasetQuery } from "metabase-types/api";
+
+export function areQueriesEqual(
+  query1: DatasetQuery,
+  query2: DatasetQuery,
+  fieldIds?: number[],
+): boolean {
+  return ML.query_EQ_(query1, query2, fieldIds);
+}
```

Comment:
Do you think we'd ever need to compare opaque pMBQL queries? If so, would it make sense to reserve `areQueriesEqual` name for them and use something like `areLegacyQueriesEqual` in this case?